Bitrise/testing_bitrise.yml

```diff
@@ -115,8 +115,8 @@ workflows:
   wetransfer_pr_testing:
     before_run:
     - pr_testing
-    after_run:
-    - workflow_danger
+#    after_run:
```

Comment:
Maybe we can add a comment here to remember to enable this again as soon as Danger supports Xcode 16